### PR TITLE
update signame&maintainner of gencomment add gitee API

### PR DESCRIPTION
**Pull request by Liran333**
1.Updated the method of get signame: identify the relationship between gitee's openeuler community org and repo
2.add gitee API to sig & maintainner
3.add welcomeMsg

### CODE REVIEW

File: robot.go [hunk 1]
```diff
@@ -245,19 +265,19 @@ func (bot *robot) getMaintainers(org, repo, sigName string, number int, config *
...
-		f, err := bot.cli.GetPathContent(config.CommunityName, config.CommunityRepo,
-			fmt.Sprintf("sig/%s/sig-info.yaml", sigName), "master")
-		if err != nil {
-			return r, nil, err
-		}
-
-		maintainers, committers := decodeSigInfoFile(*f.Content)
-		return maintainers.UnsortedList(), committers.UnsortedList(), nil
-	}
+	//_, err = bot.cli.GetPathContent(config.CommunityName, config.CommunityRepo,
```
Comment (yangwei999) [inline robot.go:268]: 不要的代码直接删除

File: main.go [hunk 2]
```diff
@@ -47,12 +54,16 @@ func main() {
 	if err := secretAgent.Start([]string{o.github.TokenPath}); err != nil {
 		logrus.WithError(err).Fatal("Error starting secret agent.")
 	}
+	c := githubclient.NewClient(secretAgent.GetTokenGenerator(o.github.TokenPath))
 
-	defer secretAgent.Stop()
+	if err := secretAgent.Start([]string{o.gitee.TokenPath}); err != nil {
```
Comment (yangwei999) [inline main.go:59]: 一个agent应该可以传入多个path

File: robot.go (hunk 1 above)
Comment (Liran333) [reply robot.go]: done

File: main.go (hunk 2 above)
Comment (Liran333) [reply main.go]: done

**Comment by yangwei999** [comment]
lgtm

**Pull request merged**